nltk_contrib/timex3.py

```diff
@@ -175,9 +171,14 @@ def ground(tagged_text, base_date):
     timex_found = timex_regex.findall(tagged_text)
     timex_found = map(lambda timex:re.sub(r'</?TIMEX2.*?>', '', timex), \
                 timex_found)
+    timexList = []
 
     # Calculate the new date accordingly
     for timex in timex_found:
+        # global month
+        month = "(january|february|march|april|may|june|july|august|september| \
+        october|november|december)"
+
```

Comment:
There was a bug in the original version here where the global variable month was overwritten and then unusable on subsequent calls to timex.  Adding month in here solves the issue.